packages/components/src/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { CSSProperties } from "react";
+import classnames from "classnames";
+import styles from "./Avatar.css";
+import { isDark } from "./utilities";
+import { Icon } from "../Icon";
+
+type AvatarSize = "base" | "large";
+
+interface AvatarProps {
+  /**
+   * A path to the image that will be displayed
+   */
+  readonly imageUrl?: string;
+  /**
+   * A users name to be used for assistive technology
+   */
+  readonly name?: string;
```

Comment:
Wonder if we should make name or initials required?

Reply:
I thought about this originally. The reason I didn't is:

- for name, this is used for accessibility and only is used if there is an `imageUrl` set
- for initials, they are not required if you set an `imageUrl`

This is also the reason that I left the user icon as a fallback.

Thoughts?

Reply:
I see what you're saying. We can do some type fanciness in here then to ensure that if we have a `url` then initials aren't required.

Eg.
https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Button/Button.tsx#L25-L43

Also happy to pair on this one if you wish. :)